### ISSUE TITLE
Reference other profiles in development

### ISSUE THREAD

**Issue by bfreeds** [Project-OSRM/osrm-backend#1524]
It was brought up at SOTMUS that linking to or including the repos for more of the profiles in development could be useful. I think it was @clhenrick

I'm not sure if the best way it through the wiki or finding a way to include those developing profiles here somehow, so I am bringing it up to discuss.

**Comment by TheMarex** [comment]
We could create a repository `osrm-custom-profile` or something. (open for a better name) My main concern is versioning: Typically people write a profile for one OSRM version and don't upgrade when we change the profile API.

**Comment by danpat** [comment]
Perhaps a simple way to do this would be to add a HTML page to the current website that people could submit pull-requests against.  A simple table with:

Who | OSRM version | Profile description

could be enough.

**Comment by daniel-j-h** [comment]
Closing this as resolved.

The question came up again in IRC today (and apparently during the 2016 SotM) and some people jumped into the discussion. I opened

https://github.com/Project-OSRM/osrm-profiles-contrib

which versions community contributed and unofficial profiles by major, minor osrm release. Should probably contain more metadata and maybe a shiny listing.

To be clear here: this is community driven and only works with contributions :)
